services/src/test/java/org/jboss/windup/web/services/rest/MigrationProjectRegisteredApplicationsEndpointTest.java

```diff
@@ -56,15 +56,32 @@ public void setUp()
 
         this.migrationProjectRegisteredApplicationsEndpoint = target.proxy(MigrationProjectRegisteredApplicationsEndpoint.class);
         this.registeredApplicationEndpoint = target.proxy(RegisteredApplicationEndpoint.class);
-        
+
         this.dummyProject = this.dataProvider.getMigrationProject();
     }
 
     @Test
     @RunAsClient
+    public void testRegisterAppFromProject() throws Exception
```

Comment:
Right, it should be named testRegister**ed**AppFromProject(). I don't like the name RegisteredApplication, too long and not much meaningful. I'd rather name the other one UserApplication.